util/testslide-snippets/README.md

```diff
@@ -0,0 +1,52 @@
+# TestSlide Snippets collection
+
+Set of snippets to help and conveniently sepeed up the writing of tests using [TestSlide](https://testslide.readthedocs.io).
+
+## Features
+
+The snippets follow some common guidelines:
+
+- All snippets are prefixed with `ts` (for TestSlide).
+- Snippets starting with `tsb_` will generate basic test cases and add a before clause.
+- Snippets starting with `tsa_` will generate basic test cases and add an after clause.
+- Snippets starting with `tsab_` will generate basic test cases with both before and after clause.
+- Snippets starting with `tsab_` will generate basic async test cases and add a before clause.
+- Snippets starting with `tsaa_` will generate basic async test cases and add an after clause.
+- Snippets starting with `tsaab_` will generate basic async test cases with both before and after clause.
```

Comment:
I think this is bound to get out of sync with snippets. How about just keep the "everything is ts*"?